docs/recipe_09.rst

```diff
@@ -0,0 +1,314 @@
+  *
+
+""""""""""""""""""""""""""""""""""""""""""""""""
+Writing an ORCA input file from a SMILES string
+""""""""""""""""""""""""""""""""""""""""""""""""
+
+- In this recipe we use the SMILES string of uridine to quickly generate an input file for geometry optimization and frequency calculation.
+- Writing molecules from SMILES requires RDKIT which can be installed with ``pip install rdkit``
+
+::
+
+    import cctk, sys
+
+    write_path = "test/static/orca_uridine_opt_freq.inp"
+
+    # define a smiles string
+    SMILES = "C1=CN(C(=O)NC1=O)C2C(C(C(O2)CO)O)O"
+
+    # define a cctk molecule from SMILES string
+    mol = cctk.Molecule.new_from_smiles(SMILES)
+
+    cctk.OrcaFile.write_molecule_to_file(write_path, mol, 
+	    header="! b3lyp/G 6-31g(d) D3 CPCM(water) opt freq tightscf",
+	    variables={"maxcore": 1000},
+	    blocks={"pal": ["nproc 4"] 
+            },
+        )
+
+This returns the following ``.inp`` file.
+
+::
+
+    ! b3lyp/G 6-31g(d) D3 CPCM(water) opt freq tightscf
+    %maxcore 1000
+    %pal
+      nproc 4
+    end
+
+    * xyz 0 1
+    6          3.81472230    0.33750522    0.14614943
+    6          2.70351934    0.51809502   -0.57053703
+    7          1.44507718    0.26401237   -0.05548295
+    6          1.27698672   -0.21741368    1.23580658
+    8          0.18983726   -0.48697883    1.74334991
+    7          2.42703629   -0.39430982    1.96055412
+    6          3.69690514   -0.15269515    1.52668941
+    8          4.68861151   -0.33199659    2.22232127
+    6          0.27652907    0.46311659   -0.91630781
+    6         -0.43325704   -0.82726794   -1.36208439
+    6         -1.75885904   -0.78542173   -0.61961108
+    6         -1.98108745    0.71387446   -0.44165945
+    8         -0.67931598    1.29470444   -0.23261185
+    6         -2.88835597    1.07023466    0.73521751
+    8         -2.50963593    0.39274159    1.93051863
+    8         -2.78649902   -1.41911137   -1.38349736
+    8         -0.67447364   -0.80085635   -2.78132772
+    1          4.80783558    0.53296161   -0.23537906
+    1          2.76381254    0.87833512   -1.59442019
+    1          2.32244110   -0.73365402    2.90389299
+    1          0.58795124    1.03504050   -1.79944336
+    1          0.13832243   -1.73754263   -1.15744293
+    1         -1.65814829   -1.31425822    0.33219174
+    1         -2.39890432    1.15967953   -1.35317397
+    1         -3.92426777    0.79417652    0.51661742
+    1         -2.84833598    2.14471841    0.94008678
+    1         -1.52917957    0.41925484    1.99037051
+    1         -3.54789329   -1.53307223   -0.78323817
+    1         -1.52137470   -1.28387237   -2.89754915
+    *
+
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Reading Simple ORCA Output Files
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Reading an ORCA output file is as simple as::
+
+  path = 'test/static/orca_uridine_opt_freq.out'
+  file = cctk.OrcaFile.read_file(path)
+
```

Comment:
The most common things people might want are (a) the last geometry and (b) the last energy.  What's the easiest way to get those?  Does get_molecule() work here?